lib/locales/da-DK.yml

```
@@ -52,6 +52,8 @@ da-DK:
         - "#{first_name} #{last_name} #{last_name}"
         - "#{first_name} #{last_name} #{last_name}"
 
+    country_code: '45'
```

```suggestion
    country_code: ['45']
```

Did this work for you as it is? Looking at the other locales, it seems the generator expects this value to be in an array 🤔

---

It's working with the sting. See 520f812f2ad12783cd52f1db16be4170469d0f46

---

ah, thank you for confirming! Do you mind adding an array just to keep the pattern? If in the future we refactor these files, it's easier if they all follow the same pattern. Thanks!

---

sure, see 02f875e79d735a2b22431743e53eaa46ded862a7

---

thanks!